app/ts/App.tsx

```diff
@@ -45,16 +47,18 @@ const UniverseForkingNotice = ({ universeForkingInformation, currentTimeInBigInt
 					(time: number) => {
 						if (universeForkingInformation.deepValue === undefined) return <></>
 						if (universeForkingInformation.deepValue.isForking === false) return <></>
+						const forkingMarketSignal = new Signal(universeForkingInformation.deepValue.forkingMarket)
+						const universeSignal = new Signal(universeForkingInformation.deepValue.universe)
 						if (time <= 0) return <>
 							The universe <b>{ getUniverseName(universeForkingInformation.deepValue.universe) } </b> has forked off.
-							A disagreement on the outcome of the market { universeForkingInformation.deepValue.forkingMarket } has caused the fork.
+							A disagreement on the outcome of the market <MarketLink address = { forkingMarketSignal } pathSignal = { pathSignal }/> has caused the fork.
 							Please use some other universe.
 						</>
 						return <>
-							The universe <b>{ getUniverseName(universeForkingInformation.deepValue.universe) }</b> is currently forking.
+							The universe <b> <UniverseLink address = { universeSignal } pathSignal = { pathSignal }/></b> is currently forking.
 							The fork will conclude in { humanReadableDateDelta(time) } ({ formatUnixTimestampIso(universeForkingInformation.deepValue.forkEndTime) }).
-							This fork was triggered by a disagreement over the outcome of the market { universeForkingInformation.deepValue.forkingMarket }.
-							Please migrate your reputation tokens before the fork ends to avoid losing them.
+							This fork was triggered by a disagreement over the outcome of the market <MarketLink address = { forkingMarketSignal } pathSignal = { pathSignal }/>.
```

Comment:
Can we display the market short description here?  Just showing an address feels rather useless, no one cares what the address is.